__tests__/app.test.js

```diff
@@ -0,0 +1,32 @@
+const { app } = require("../app.js");
+const connection = require("../db/connection.js");
+const request = require("supertest");
+const seed = require("../db/seeds/seed.js");
+const testData = require("../db/data/test-data");
+
+beforeEach(() => seed(testData));
+
+afterAll(() => {
+  connection.end();
+});
+describe("GET api/topics", () => {
+  it("responds with array containing all topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.topics.length).toBe(3);
+      });
+  });
+  it("should have correct properties for each topic", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.topics[0]).toHaveProperty(
```

Comment:
I would suggest using a forEach here rather than just checking the first object - just to ensure all objects have these properties :)